libs/client/client.go

```diff
@@ -43,6 +46,7 @@ func NewDefaultTestNetClient(veloNodeUrl string, stellarAccountSecretKey string)
 	return NewTestNetClient(grpcConn, stellarAccountSecretKey)
 }
 
+// New public client is a custom client to connect to velo public network.
```

Comment:
NewPublicClient creates a new public network client instance to with custom GRPC connection